docs/README.md

```diff
@@ -11,149 +11,59 @@ Each bag is identified with a **space** (a broad category) an **external identif
-It does not do any introspection of the bag contents, or change its behaviour based on the files a bag contains.
-
-
-
-## Read more
-
-If you'd like to learn more about the storage service:
-
--   [**Building Wellcome Collection’s new archival storage service**](https://stacks.wellcomecollection.org/building-wellcome-collections-new-archival-storage-service-3f68ff21927e) explains more of the ideas and principles behind the storage service, and how we successfully migrated all of our digitised content into the system.
-
--   [**How we store multiple versions of BagIt bags**](https://stacks.wellcomecollection.org/how-we-store-multiple-versions-of-bagit-bags-e68499815184) explains how we've added versioning atop the BagIt spec, to track distinct versions of bags.
-
-We've also written about some of the code used in the storage service:
-
--   [**Creating a locking service in a Scala type class**](https://alexwlchan.net/2019/05/creating-a-locking-service-in-a-scala-type-class/).
-    When we're writing a bag to an S3 bucket, we want to be sure that only one process is running at a time -- we don't want multiple processes writing to the same bucket, and potentially writing conflicting data.
-    Because S3 doesn't have support for this sort of locking, we have to implement it in our code.
-
--   [**Iterating over the entries of a compressed archive (tar.gz)**](https://alexwlchan.net/2019/09/unpacking-compressed-archives-in-scala/) (Scala).
-    When a user uploads a bag, it has to be packaged as a tar.gz file.
-    This explains the code we use to unpack those archives.
-
--   [**Streaming large objects from S3 with ranged GET requests**](https://alexwlchan.net/2019/09/streaming-large-s3-objects/) (Java/Scala).
-    If you're trying to read a large object from S3, the connection may drop before you read all the bytes.
-    Using ranged GET requests allows you to request a piece at a time, and then you can use Java's `SequenceInputStream` to combine them together.
-
--   [**Working with really large objects in S3**](https://alexwlchan.net/2019/02/working-with-large-s3-objects/) (Python).
-    A wrapper class that adds a couple of methods to the file-like objects returned by the AWS SDK for Python, allowing you to use it with the zipfile module and the like.
-
-
-
-## Issues
-
-You can see a list of open issues for the storage service [in the wellcomecollection/platform repo](https://github.com/wellcomecollection/platform/issues?q=is%3Aissue%20is%3Aopen%20label%3A%22%F0%9F%93%A6%20Storage%20service%22).
-
+## Reference
 
-## Releasing
+These topics explain how the storage service work, and why it's designed in the way it is:
 
-```sh
-# Ensure you have the correct AWS credentials in scope
-# before running!
+-   [Detailed architecture: what do the different services do?]
+-   [How identifiers work in the storage service](explanations/identifiers.md)
+-   [How files are laid out in the underlying storage](explanations/file-layout.md)
```

Comment:
I don't think `explanations/file-layout.md` is in this PR.

Reply:
Oops, probably because I haven't written it yet. That's next.